### ISSUE TITLE
Logstash: Encountered a retryable error code=>403

### ISSUE THREAD

**Issue by alexanderfefelov** [alexanderfefelov/docker-backpack#103]
```
[2021-06-11T14:06:06,193][ERROR][logstash.outputs.elasticsearchmonitoring][.monitoring-logstash][369ae0f595849b3d26feac1c4bc25b6c5cdd3b1d0ab7c7d6c9548bb5ac76bfdd] Encountered a retryable error (will retry with exponential backoff) {:code=>403, :url=>"http://elasticsearch.backpack.test:9200/_monitoring/bulk?system_id=logstash&system_api_version=7&interval=1s", :content_length=>1390}
```

**Comment by alexanderfefelov** [comment]
https://github.com/elastic/logstash/issues/7905

**Comment by alexanderfefelov** [comment]
Fixed by b09ea4f
